tools/data_log_check/radio_mod_check.py

```diff
@@ -0,0 +1,103 @@
+import struct
+import sys
+
+GNSS_DATA_FORMAT = "fffIIBBHB"
```

Comment:
What does this mean?

Reply:
https://docs.python.org/3/library/struct.html#format-characters